src/services/navigationBar.ts

```diff
@@ -1,8 +1,217 @@
+/// <reference path='utilities.ts' />
 /// <reference path='services.ts' />
 
 /* @internal */
 namespace ts.NavigationBar {
+    export function getJsNavigationBarItems(sourceFile: SourceFile, compilerOptions: CompilerOptions): NavigationBarItem[] {

```

Comment:
Can you move this *below* `getNavigationBarItems`?Generally we have helper functions cascading below the entry point.